src/ray/common/task/task_spec.cc

```diff
@@ -189,16 +189,10 @@ bool TaskSpecification::IsRetry() const { return AttemptNumber() > 0; }
 int32_t TaskSpecification::MaxRetries() const { return message_->max_retries(); }
 
 int TaskSpecification::GetRuntimeEnvHash() const {
-  absl::flat_hash_map<std::string, double> required_resource;
-  if (RayConfig::instance().worker_resource_limits_enabled()) {
```

Comment:
What's the story behind this feature flag? Is it very old and just tech debt?

Reply:
i think so. probably can be removed.